### DISCUSSION TITLE
[Question] How to show some numbers besides robots instances, such as major joint position

### DISCUSSION THREAD

**Discussion by 47449345** [isaac-sim/IsaacLab#508] [Q&A]
Hi,

I want show some numbers besides robots, such as major joint position, motion phase, for keep watching the robot status.

I noticed the marker, but it's just some shape, is there any way to show numbers and words as label ?

thanks.

**Comment by Mayankm96** [comment]
Where do you want to show them?

**Comment by 47449345** [comment]
![pic](https://github.com/NVIDIA-Omniverse/orbit/assets/26254094/52f29a5d-3be7-4e96-8f2e-a896a86fb91c)

Just like the attached pic, show some labels, and can be updated on every step.

**Comment by 47449345** [comment]
@Mayankm96 Please check this.

**Comment by Mayankm96** [comment]
Maybe these help:

* https://forums.developer.nvidia.com/t/display-text-on-the-render-viewport/174675/6
* https://github.com/NVIDIA-Omniverse/kit-extension-sample-ui-scene/tree/main/exts/omni.example.ui_scene.widget_info
